frontend/components/common/AppProviders.tsx

```diff
@@ -2,6 +2,7 @@ import { ChakraProvider, extendTheme } from '@chakra-ui/react';
 import { IFIXIT_ORIGIN } from '@config/env';
 import { AppProvider } from '@ifixit/app';
 import { theme } from '@ifixit/ui';
+import { CartDrawerProvider } from '@ifixit/ui/cart/drawer/hooks/useCartDrawer';
```

Comment:
This might be personal preference, but we often export deeply nested files from a parent container. Here, we might `export * from './drawer/hooks/useCartDrawer';` in `packages/ui/cart/index.ts`, and then we can import here like

```suggestion
import { CartDrawerProvider, theme } from '@ifixit/ui';
```